config/application.yml.default

```diff
@@ -503,6 +505,7 @@ production:
   skip_encryption_allowed_list: '["urn:gov:gsa:SAML:2.0.profiles:sp:sso:dev", "urn:gov:gsa:SAML:2.0.profiles:sp:sso:int"]'
   state_tracking_enabled: false
   telephony_adapter: pinpoint
+  use_fed_domain_class: false
```

Comment:
Noting that this will cause an error if #11082 is merged before this. We could address the issue by only overriding where it's different from default.